.github/workflows/build.yml

```diff
@@ -60,6 +60,9 @@ jobs:
     steps:
     - name: Check out code
       uses: actions/checkout@v1
+    - name: go-check
+      run: |
+        make check
```

Comment:
This should run with skipper
Otherwise we get different minors of go, and we've seen this to cause issues.
Specifically with go-import
Notice that this should be after the step installing python and skipper

Reply:
Done